src/ORM/Table.php

```diff
@@ -700,14 +700,27 @@ public function setDisplayField($field)
     public function getDisplayField()
     {
         if ($this->_displayField === null) {
+            $displayFieldFound = false;
             $schema = $this->getSchema();
             $this->_displayField = $this->getPrimaryKey();
-            foreach (['title', 'name', 'label'] as $field) {
+            $expectedFields = ['title', 'name', 'label'];
+            foreach ($expectedFields as $field) {
                 if ($schema->hasColumn($field)) {
                     $this->_displayField = $field;
+                    $displayFieldFound = true;
                     break;
                 }
             }
+            if (!$displayFieldFound) {
+                foreach ($expectedFields as $field) {
+                    foreach ($schema->columns() as $column) {
+                        if (substr($column, -strlen('_' . $field)) === '_' . $field) {
+                            $this->_displayField = $column;
+                            break 2;
```

Comment:
Don't see this everyday :smile:

Reply:
We don't use `break 2` much :smile: I don't think there is a better solution here though without adding more function calls.